CHANGELOG.md

```diff
@@ -79,6 +102,35 @@
   clearly labeled `experimental`.  Regardless, we apologize if this causes you
   inconvenience or additional work.
 
+**OTHER CHANGES**:
+
+* fix(storage): eliminate dangling function references ([#7366](https://github.com/googleapis/google-cloud-cpp/pull/7366))
+* doc: add gRPC steps to storage quickstart ([#7358](https://github.com/googleapis/google-cloud-cpp/pull/7358))
+* fix(storage): restore deprecation warnings on MSVC ([#7325](https://github.com/googleapis/google-cloud-cpp/pull/7325))
+* fix(storage): build with MSVC+x86 ([#7323](https://github.com/googleapis/google-cloud-cpp/pull/7323))
+* feat(GCS+gRPC): reduce data copying in downloads ([#7303](https://github.com/googleapis/google-cloud-cpp/pull/7303))
+* feat(GCS+gRPC): support timeouts for all requests ([#7299](https://github.com/googleapis/google-cloud-cpp/pull/7299))
+* feat(storage): support timeouts for all requests ([#7295](https://github.com/googleapis/google-cloud-cpp/pull/7295))
+* feat(storage): use generic parameters when resuming uploads ([#7292](https://github.com/googleapis/google-cloud-cpp/pull/7292))
+* feat(storage): support base URLs with existing query parameters ([#7291](https://github.com/googleapis/google-cloud-cpp/pull/7291))
+* feat(GCS+gRPC): implement standard parameters ([#7272](https://github.com/googleapis/google-cloud-cpp/pull/7272))
+* fix(storage): use CA info options in credentials ([#7261](https://github.com/googleapis/google-cloud-cpp/pull/7261))
+* feat(storage)!: move GCS+gRPC to v2/ protos ([#7254](https://github.com/googleapis/google-cloud-cpp/pull/7254))
+* doc(storage): handle kNotFound in example ([#7252](https://github.com/googleapis/google-cloud-cpp/pull/7252))
+
+### [Spanner](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/spanner/README.md)
+
+* feat(spanner): add factory functions for instance/database/backup ([#7321](https://github.com/googleapis/google-cloud-cpp/pull/7321))
+* doc(spanner): convert Spanner samples to use the generated admin APIs ([#7311](https://github.com/googleapis/google-cloud-cpp/pull/7311))
+* feat(spanner): add generated admin APIs ([#7285](https://github.com/googleapis/google-cloud-cpp/pull/7285))
+
+### [Common Libraries](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/README.md)
+
+* feat(generator): add "generator" to API client header "gccl" semver ([#7383](https://github.com/googleapis/google-cloud-cpp/pull/7383))
+* fix: substitute build metadata (git commit) in bazel builds ([#7378](https://github.com/googleapis/google-cloud-cpp/pull/7378))
+* feat(common): add ability to supply a user-run CQ to gRPC options ([#7354](https://github.com/googleapis/google-cloud-cpp/pull/7354))
+* feat(generator): generate a SetIamPolicy() overload with an OCC loop ([#7276](https://github.com/googleapis/google-cloud-cpp/pull/7276))
```

Comment:
No action required: did this change the generated code for IAM or BigQuery?  If so, how do we document such changes in the future?  Once for all "generated libraries"?  Or specifically for each generated library?

As a customer using IAM or BigQuery: how do I find out about the changes for the library I care about?

Reply:
OK.  I removed this "feat(generator)" from "Common" and added it to each of "BigQuery", "IAM", and "Spanner".

I also removed the "feat(generator)" about the `X-Goog-Api-Client` header, as it is really internal-only.